internal/handlers/tigris/msg_update.go

```diff
@@ -196,9 +196,9 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 
 // update updates documents by _id.
 func (h *Handler) update(ctx context.Context, sp fetchParam, doc *types.Document) (int, error) {
-	id := must.NotFail(doc.Get("_id")).(types.ObjectID)
-	f := must.NotFail(filter.Eq("_id", tjson.ObjectID(id)).Build())
-	h.L.Sugar().Debugf("Filter: %s", f)
+	id := must.NotFail(doc.Get("_id"))
+	f := must.NotFail(filter.Eq("_id", id).Build())
```

Comment:
I really think that having a separate package with three files, types, functions, builders, etc is too much for a single line: `json.Marshal(map[string]any{"_id": id})`

Let's keep it for now, but I think we should really simplify it when we will work on query pushdown